Unit1/Module0/Mini-Project0.1/README.md

```diff
@@ -4,11 +4,9 @@ This mini-project is meant to get you familiar with the assignment submission an
 
 ## Project Instructions
 
-1. Go to your Web Dev Fundamentals repository (find it by going to My Respositories in your GitHub account).
```

Comment:
I now see that "My Repositories" wouldnt have the student repo since it wasnt created by them. But I suggest we add a step for them to go to https://github.com/shift-up/ and then find the repository with their name on it.